packages/api-bindings/src/graphql/client.graphql

```diff
@@ -24,6 +26,14 @@ extend type Mirror {
   isOptimisticMirroredByMe: Boolean! # just to keep Publication type consistent, that will never be used
 }
 
+extend type Erc20Amount {
+  asAmount: ClientErc20Amount!
+}
+
+extend type ModuleFeeAmount {
+  asAmount: ClientErc20Amount!
+}
```

Comment:
Since it was almost free to add asAmount to `ModuleFeeAmount` i decided to add it here also, i think it's a good idea to have it wrapped in the class since it's going to be useful for the future (for example for `useApproveModule`  or `useFollow`), and it's easy to apply it now than later since it'll need more refactor.